modules/mongodb/mongodb_test.go

```diff
@@ -76,3 +78,38 @@ func ExampleRunContainer_connect() {
 	// Output:
 	// test
 }
+
+func ExampleRunContainer_withCredentials(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := mongodb.RunContainer(ctx,
+		testcontainers.WithImage("mongo:6"),
+		mongodb.WithUsername("root"),
+		mongodb.WithPassword("password"),
+		testcontainers.WithWaitStrategy(wait.ForLog("Waiting for connections")),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	connStr, err := container.ConnectionString(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mongoClient, err := mongo.Connect(ctx, options.Client().ApplyURI(connStr))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = mongoClient.Ping(ctx, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
A testable example needs an output comment for something already printed with fmt. Please see https://github.com/testcontainers/testcontainers-go/pull/1910/files#diff-cbb99f9ae72ba7f7b2a251421a76419f12d724152c813a69fa40b2c4d4b730cfR76 and https://github.com/testcontainers/testcontainers-go/pull/1910/files#diff-cbb99f9ae72ba7f7b2a251421a76419f12d724152c813a69fa40b2c4d4b730cfR78

Therefore this example needs both to be runnable at the Ci

Reply:
Oh, I missed that. I updated it. I also removed the testing.T argument since testable examplea do not take  arguments.